Sources/ComposableCoreMotion/HeadphoneMotionManager/HeadphoneMotionManagerInterface.swift

```diff
@@ -0,0 +1,95 @@
+import ComposableArchitecture
+import CoreMotion
+
+/// A wrapper around Core Motion's `CMHeadphoneMotionManager` that exposes its functionality through
+/// effects and actions, making it easy to use with the Composable Architecture, and easy to test.
```

Comment:
should we add more docs here about usage like we've done with other support libraries?

Reply:
Yeah...I tried making a demo but it looks like this API doesn't even work yet: https://developer.apple.com/forums/thread/650051

Converted to a draft that we can revisit in later betas.